synapse/lib/stormtypes.py

```diff
@@ -452,11 +453,24 @@ def _reqStr(self, name):
                 ret.append((key, valu))
         return ret
 
+class NodeData(Prim):
+
+    def __init__(self, node, path=None):
+        Prim.__init__(self, node, path=path)
+
+        async def listdata():
+            return [x async for x in node.iterData()]
+
+        self.locls.update({
+            'get': node.getData,
+            'set': node.setData,
```

Comment:
it feels like there is fundamentally a `$node.data.del()` operation missing here.